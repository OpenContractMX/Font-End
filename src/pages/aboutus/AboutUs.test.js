import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { AboutUs } from "./";

it("[render component `pages / AboutUs`]", () => {
  const { asFragment } = render(<AboutUs />);
  expect(asFragment()).toMatchSnapshot();
});
