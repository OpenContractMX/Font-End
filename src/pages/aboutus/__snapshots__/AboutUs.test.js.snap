// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[render component \`pages / AboutUs\`] 1`] = `
<DocumentFragment>
  <main
    class="about-us"
  >
    <section
      class="about-us__container"
    >
      <h2
        class="about-us__container--title"
      >
        Equipo técnico
      </h2>
      <article
        class="card-presentation"
      >
        <figure
          class="card-presentation__image-card"
        >
          <img
            alt=""
            src="leo.jpg"
          />
           
        </figure>
        <p
          class="card-presentation__title-name"
        >
          Leonardo Rincón
        </p>
        <p
          class="card-presentation__title-job-title"
        >
          Frontend Developer
        </p>
        <div
          class="card-presentation__container-icons"
        >
          <span
            class="card-presentation__container-icons-linkedin-icon"
          >
            <a
              href="https://www.linkedin.com/in/leo-rincon/"
              target="_blank"
            >
              <i
                class="fab fa-linkedin-in"
              />
            </a>
          </span>
          <span
            class="card-presentation__container-icons-twitter-icon"
          >
            <a
              href="https://twitter.com/LeoBotache7"
              target="_blank"
            >
              <i
                class="fab fa-twitter"
              />
            </a>
          </span>
        </div>
      </article>
      <article
        class="card-presentation"
      >
        <figure
          class="card-presentation__image-card"
        >
          <img
            alt=""
            src="jc.jpg"
          />
           
        </figure>
        <p
          class="card-presentation__title-name"
        >
          Juan Camilo Garcés
        </p>
        <p
          class="card-presentation__title-job-title"
        >
          Frontend Developer
        </p>
        <div
          class="card-presentation__container-icons"
        >
          <span
            class="card-presentation__container-icons-linkedin-icon"
          >
            <a
              href="https://www.linkedin.com/in/juancamilogarcesviveros/"
              target="_blank"
            >
              <i
                class="fab fa-linkedin-in"
              />
            </a>
          </span>
          <span
            class="card-presentation__container-icons-twitter-icon"
          >
            <a
              href="https://twitter.com/Camilo_GarcesV"
              target="_blank"
            >
              <i
                class="fab fa-twitter"
              />
            </a>
          </span>
        </div>
      </article>
      <article
        class="card-presentation"
      >
        <figure
          class="card-presentation__image-card"
        >
          <img
            alt=""
            src="enrique.jpeg"
          />
           
        </figure>
        <p
          class="card-presentation__title-name"
        >
          Enrique Pérez
        </p>
        <p
          class="card-presentation__title-job-title"
        >
          Data Scientist/Backend Developer
        </p>
        <div
          class="card-presentation__container-icons"
        >
          <span
            class="card-presentation__container-icons-linkedin-icon"
          >
            <a
              href="https://www.linkedin.com/in/enriqueeperez/"
              target="_blank"
            >
              <i
                class="fab fa-linkedin-in"
              />
            </a>
          </span>
          <span
            class="card-presentation__container-icons-twitter-icon"
          >
            <a
              href="https://twitter.com/EnriqueePerez"
              target="_blank"
            >
              <i
                class="fab fa-twitter"
              />
            </a>
          </span>
        </div>
      </article>
      <article
        class="card-presentation"
      >
        <figure
          class="card-presentation__image-card"
        >
          <img
            alt=""
            src="jorge.jpg"
          />
           
        </figure>
        <p
          class="card-presentation__title-name"
        >
          Jorge Alberto Delgadillo Alonso
        </p>
        <p
          class="card-presentation__title-job-title"
        >
          Data Scientist/Backend Developer
        </p>
        <div
          class="card-presentation__container-icons"
        >
          <span
            class="card-presentation__container-icons-linkedin-icon"
          >
            <a
              href="https://www.linkedin.com/in/jorge-alberto-delgadillo-alonso-7b58501aa/"
              target="_blank"
            >
              <i
                class="fab fa-linkedin-in"
              />
            </a>
          </span>
          <span
            class="card-presentation__container-icons-twitter-icon"
          >
            <a
              href="https://twitter.com/jorge64311416"
              target="_blank"
            >
              <i
                class="fab fa-twitter"
              />
            </a>
          </span>
        </div>
      </article>
    </section>
  </main>
</DocumentFragment>
`;
